config/locales/en.yml

```diff
@@ -48,14 +48,14 @@ en:
         zero: external activities
         one: external activity
         other: external activities
-      Interactive: Interactive
+      Interactive: Model
       interactive:
-        zero: interactives
-        one: interactive
-        other: interactives
+        zero: models
+        one: model
+        other: models
   investigation: sequence
   activity: activity
-  interactive: interactive
+  interactive: model
   material: material
   instructional_materials: Instructional Materials
   recent_activity:
```

Comment:
In this file we also need to add the mapping of interactive to interactive in the itsi-learn theme.